code_test.go

```diff
@@ -90,3 +90,32 @@ func TestFlagValue(t *testing.T) {
 	}
 
 }
+
+func TestKnownCodes(t *testing.T) {
+	t.Parallel()
+
+	codes := multicodec.KnownCodes()
+	if len(codes) < 100 {
+		t.Fatalf("expected list to have at least 100 elements")
+	}
+	seen := map[multicodec.Code]bool{}
+	missing := map[multicodec.Code]bool{
```

Comment:
this isn't 'missing' so much as 'expected'?

Reply:
I actually thought about this for half a minute :) it's "expected" when declared, and "missing" when used at the end of the function. so I found "missing" more intuitive: we declare it, update it via the loop, and are left with... the missing bits.